app/views/pallets/index.html.erb

```diff
@@ -22,10 +22,10 @@
   <tbody>
     <% @pallets.each do |pallet| %>
       <tr>
-        <td><%= pallet.reception %></td>
+        <td><%= pallet.reception.id %></td>
         <td><%= pallet.pallet_number %></td>
         <td><%= pallet.product.code %></td>
-        <td><%= pallet.location %></td>
+        <td><%= pallet.location.location_full_name %></td>
```

Comment:
you must use `try(:nested_model)` to avoid exceptions caused when the object hasn't the nested model.

an example in this case::
`pallet.try(:location).location_full_name`

Reply:
but a pallet will allways have a Location, then i don't need use try

Reply:
then add a `validates :location_id, presence: true in the model`

Reply:
ok boss